TLM/TLM/Manager/Impl/ExtUITabStrip.cs

```diff
@@ -0,0 +1,151 @@
+            panel.autoLayout = true;
+            panel.autoLayoutDirection = LayoutDirection.Horizontal;
+
+            UIScrollablePanel scrollablePanel = panel.AddUIComponent<UIScrollablePanel>();
+            scrollablePanel.autoLayout = true;
+            scrollablePanel.autoLayoutStart = LayoutStart.TopLeft;
+            scrollablePanel.wrapLayout = true;
+            scrollablePanel.size = new Vector2(panel.size.x - 50, panel.size.y + 35);
+            scrollablePanel.autoLayoutDirection = LayoutDirection.Horizontal; //Vertical does not work but why?
+
+            UIScrollbar verticalScrollbar = CreateVerticalScrollbar(panel, scrollablePanel);
+            verticalScrollbar.Show();
+            verticalScrollbar.Invalidate();
+            scrollablePanel.Invalidate();
+
+            return scrollablePanel;
+        }
+
+        public UIHelper AddTabPage(string name, bool scrollBars=true) {
+            UIButton tabButton = base.AddTab(name);
+            tabButton.normalBgSprite = "SubBarButtonBase";
+            tabButton.disabledBgSprite = "SubBarButtonBaseDisabled";
+            tabButton.focusedBgSprite = "SubBarButtonBaseFocused";
+            tabButton.hoveredBgSprite = "SubBarButtonBaseHovered";
+            tabButton.pressedBgSprite = "SubBarButtonBasePressed";
+            tabButton.textPadding = new RectOffset(10, 10, 10, 6);
+            tabButton.autoSize = true;
+
+            selectedIndex = tabCount - 1;
+            UIPanel currentPanel = tabContainer.components[selectedIndex] as UIPanel;
+            currentPanel.autoLayoutPadding = new RectOffset(10, 10, 0, 16);
+            currentPanel.autoLayout = true;
+
+            UIHelper panelHelper;
+            if (scrollBars) {
+                UIScrollablePanel scrollablePanel = CreateScrollebalePanel(currentPanel);
+                panelHelper = new UIHelper(scrollablePanel);
+            }
+            else {
+                currentPanel.autoLayoutDirection = LayoutDirection.Vertical;
+                panelHelper = new UIHelper(currentPanel);
+            }
+            return panelHelper;
+        }
+
+        public static ExtUITabstrip Create(UIHelperBase helper_base) {
+            UIHelper actualHelper = helper_base as UIHelper;
+            UIComponent container = actualHelper.self as UIComponent;
+
+            ExtUITabstrip tabStrip = container.AddUIComponent<ExtUITabstrip>();
+            tabStrip.relativePosition = new Vector3(0, 0);
+            tabStrip.size = new Vector2(container.width - 20, 40);
+
+            float h = container.height - tabStrip.height * 2 - 40;
+            UITabContainer tabContainer = container.AddUIComponent<UITabContainer>();
+            tabContainer.relativePosition = new Vector3(0, 80);
+            tabContainer.width = tabStrip.width;
+            tabContainer.height = h;
+            tabStrip.tabPages = tabContainer;
+
+            return tabStrip;
+        }
+
+        public static class Test {
```

Comment:
This static class should be encapsulated in `#if DEBUG ... #endif` block? I don't see any need for it to be in `RELEASE` or `LABS` builds.